react/src/components/Weather/Forecast/Forecast.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import {
+    Accordion,
+    AccordionItem,
+    AccordionItemHeading,
+    AccordionItemButton,
+    AccordionItemPanel,
+} from "react-accessible-accordion";
+import ForecastStyle from './Forecast.module.css';
+import sun from '../../../icons/desktop/weathersun.png';
+import rain from '../../../icons/desktop/weatherrain.png';
+import cloud from '../../../icons/desktop/weathercloud.png';
+
+const ICONS = {
+    'Clear': sun,
+    'Rain': rain,
+    'Clouds': cloud
+};
+
+const WEEK_DAYS = ['Poniedziałek', 'Wtorek', 'Środa', 'Czwartek', 'Piątek', 'Sobota', 'Niedziela'];
+
+const Forecast = ({ data }) => {
+    const dayInAWeek = new Date().getDay();
+    const forecastDays = WEEK_DAYS.slice(dayInAWeek, WEEK_DAYS.length).concat(WEEK_DAYS.slice(0, dayInAWeek));
+    console.log("data", data)
+    if(!data) return null;
+
+    return (
+        <>
+            <label className={ForecastStyle.title}>Pogoda długoterminowa</label>
+            <Accordion allowZeroExpanded className={ForecastStyle.container}>
+                {data?.list.splice(0, 7).map((item, idx) => (
+                    <AccordionItem key={idx}>
+                        <AccordionItemHeading>3
+                            <AccordionItemButton className={ForecastStyle.wrapper}>                           
+                                <li className={ForecastStyle.daily_item}>
+                                    <div className={ForecastStyle.weather}>
+                                        <img src={ICONS[item.weather[0].main]} />
+                                    </div>
+                                    <label className={ForecastStyle.day}>{forecastDays[idx]}</label>
+                                    <label className={ForecastStyle.min_max}>{Math.round(item.main.temp)}°C</label>
+                                </li>
+                            </AccordionItemButton>
+                        </AccordionItemHeading>
+                        <AccordionItemPanel className={ForecastStyle.wrapper}>
+                            <div className={ForecastStyle.daily_details_grid}>
+                                <div className={ForecastStyle.daily_details_grid_item}>
+                                    <label>Pressure:</label>
```

Comment:
label służy tylko w połączeniu z inputami, tutaj bym to na coś innego zamienił

Reply:
@ssmolenn  Komentarz powinien zamknąć autor. Jeśli jest na prawdę dobry powód, dla którego zamknąłeś, to zawsze trzeba dodać to w komentarzu